src/openrpc/lifecycle.json

```diff
@@ -1,379 +1,390 @@
+                    }
+                },
+                {
+                    "name": "Close the app when the user selects an exit menu item",
+                    "params": [
+                        {
+                            "name": "reason",
+                            "value": "userExit"
+                        }
+                    ],
+                    "result": {
+                        "name": "Default Result",
+                        "value": null
+                    }
+                }
+            ]
         },
         {
-          "name": "capabilities",
-          "x-uses": ["xrn:firebolt:capability:lifecycle:state"]
-        }
-      ],
-      "summary": "Notify the platform that the app is done unloading",
-      "params": [],
-      "result": {
-        "name": "results",
-        "schema": {
-          "const": null
-        }
-      },
-      "examples": [
-        {
-          "name": "Default Example",
-          "params": [],
-          "result": {
-            "name": "Default Result",
-            "value": null
-          }
-        }
-      ]
-    },
-    {
-      "name": "state",
-      "summary": "Get the current state of the app. This function is **synchronous**.",
-      "tags": [
-        {
-          "name": "synchronous"
+            "name": "start",
+            "tags": [
+                {
+                    "name": "notifier",
+                    "x-event": "Lifecycle.onStart"
+                },
+                {
+                    "name": "capabilities",
+                    "x-uses": [
+                        "xrn:firebolt:capability:lifecycle:state"
+                    ]
+                }
+            ],
+            "summary": "Listen to the start event",
+            "params": [],
+            "examples": [
+                {
+                    "name": "Default Example",
+                    "params": [],
+                    "result": {
+                        "name": "Default Result",
+                        "value": {
+                            "state": "paused",
+                            "previous": "initializing"
+                        }
+                    }
+                }
+            ]
         },
         {
-          "name": "exclude-from-sdk"
+            "name": "startSuspend",
+            "tags": [
+                {
+                    "name": "notifier",
+                    "x-event": "Lifecycle.onStartSuspend"
+                },
+                {
+                    "name": "capabilities",
+                    "x-uses": [
+                        "xrn:firebolt:capability:lifecycle:state"
+                    ]
+                }
+            ],
+            "summary": "Listen to the suspended start event",
+            "params": [],
+            "examples": [
+                {
+                    "name": "Default Example",
+                    "params": [],
+                    "result": {
+                        "name": "Default Result",
```

Comment:
notifiers have no result. Does this pass validation via `firebolt-openrpc`? If so we should fix that.